docker-compose.backend.master.yml

```diff
@@ -27,3 +27,7 @@ services:
       service: redis_cache
     environment:
       - KOBO_REDIS_SERVER_ROLE=cache
+
+networks:
+  kobo-be-network:
+    driver: bridge
```

Comment:
What does this change?

Reply:
@jnm If you look at `docker-compose.frontend.override.yml.sample`, you can see that `kpi`, `kc` and `ee` are also using this network. Because backend and frontend containers are not declared in the same composer file, frontend containers need to know what is the name of backend containers network (when ports are not exposed).